assets/config.php

```diff
@@ -3,113 +3,287 @@
     ],
+
+    /**
+     * TenantDatabaseManagers are classes that handle the creation & deletion of tenant databases.
+     */
     'database_managers' => [
-        // Tenant database managers handle the creation & deletion of tenant databases.
         'sqlite' => Stancl\Tenancy\TenantDatabaseManagers\SQLiteDatabaseManager::class,
         'mysql' => Stancl\Tenancy\TenantDatabaseManagers\MySQLDatabaseManager::class,
         'pgsql' => Stancl\Tenancy\TenantDatabaseManagers\PostgreSQLDatabaseManager::class,
+
+        /**
+         * Disable the pgsql manager above, enable the one below, and set the
+         * tenancy.database.separate_by config key to 'schema' if you would
+         * like to separate tenant DBs by schemas rather than databases.
+         */
         // 'pgsql' => Stancl\Tenancy\TenantDatabaseManagers\PostgreSQLSchemaManager::class, // Separate by schema instead of database
     ],
+
+    /**
+     * Connections used by TenantDatabaseManagers. This tells, for example, the
+     * MySQLDatabaseManager to use the mysql connection to create databases.
+     */
     'database_manager_connections' => [
-        // Connections used by TenantDatabaseManagers. This tells, for example, the
-        // MySQLDatabaseManager to use the mysql connection to create databases.
         'sqlite' => 'sqlite',
         'mysql' => 'mysql',
         'pgsql' => 'pgsql',
     ],
-    'bootstrappers' => [
-        // Tenancy bootstrappers are executed when tenancy is initialized.
-        // Their responsibility is making Laravel features tenant-aware.
-        'database' => Stancl\Tenancy\TenancyBootstrappers\DatabaseTenancyBootstrapper::class,
-        'cache' => Stancl\Tenancy\TenancyBootstrappers\CacheTenancyBootstrapper::class,
-        'filesystem' => Stancl\Tenancy\TenancyBootstrappers\FilesystemTenancyBootstrapper::class,
-        'queue' => Stancl\Tenancy\TenancyBootstrappers\QueueTenancyBootstrapper::class,
-        // 'redis' => Stancl\Tenancy\TenancyBootstrappers\RedisTenancyBootstrapper::class, // Note: phpredis is needed
-    ],
+
+    /**
+     * Features are classes that provide additional functionality
+     * not needed for tenancy to be bootstrapped. They are run
+     * regardless of whether tenancy has been initialized.
+     *
+     * See the documentation page for each class to
+     * understand which ones you want to enable.
+     */
     'features' => [
-        // Features are classes that provide additional functionality
-        // not needed for tenancy to be bootstrapped. They are run
-        // regardless of whether tenancy has been initialized.
-
-        // Stancl\Tenancy\Features\Timestamps::class,
-        // Stancl\Tenancy\Features\TenantConfig::class,
-        // Stancl\Tenancy\Features\TelescopeTags::class,
-        // Stancl\Tenancy\Features\TenantRedirect::class,
+        // Stancl\Tenancy\Features\Timestamps::class, // https://tenancy.samuelstancl.me/docs/v2/features/timestamps/
```

Comment:
There is a hidden setting on the code base `timestamp_format` that could be mentioned here - the thing is that if only makes sense if you enable the Timestams feature.

Reply:
Don't want to bloat the config too much, so I think I'll just mention this on the docs page. https://tenancy.samuelstancl.me/docs/v2/features/timestamps/#timestamps-redirect

Reply:
I agree that the config file being "clean" like you suggest here is great. But having config options scattered throughout the docs has its downsides as well. I sometimes wish documentation sites had a config example with every option and clear explanations of what each option does. This has the benefit of seeing all options available with a quick explanation (and maybe a link to the section of the docs for more detail) while still having a clean config file in the codebase.

How would you feel about something like that?